Dockerfile

```diff
@@ -69,10 +69,8 @@ RUN tar xzf salmon-${SALMON_VERSION}_linux_x86_64.tar.gz && \
     ln -s /usr/local/src/salmon-latest_linux_x86_64/bin/salmon /usr/local/bin/salmon
 
 # Use renv for R packages
-ENV RENV_VERSION 0.17.2
 ENV RENV_CONFIG_CACHE_ENABLED FALSE
-RUN R -e "install.packages('remotes')"
-RUN R -e "remotes::install_version('renv', version = '${RENV_VERSION}')"
+RUN R -e "install.packages(c('renv', 'yaml', 'BiocManager'))"
```

Comment:
curious why no `remotes`? I'm assuming this is all fine though, so will approve to not hold up.

Reply:
Because I was trying to test a bunch of things, and removing `install_version` was one of those. `remotes` will still get installed by `renv` (as will the other things here, but as I said, I was testing)